src/liballoc/sync.rs

```diff
@@ -107,7 +108,7 @@ const MAX_REFCOUNT: usize = (isize::MAX) as usize;
 /// // The two syntaxes below are equivalent.
 /// let a = foo.clone();
 /// let b = Arc::clone(&foo);
-/// // a and b both point to the same memory location as foo.
+/// // a and b both point to the same memory location as foo
```

Comment:
If this still sounds confusing you could say "a and b both point to the same memory location as foo _does_" or "a, b, and foo all point to the same memory location"

Reply:
Makes sense, I probably should make that change to make it even clearer. I know for a fact that folks with English as a second language do appreciate the extra wording.